.bldr.toml

```diff
@@ -513,15 +513,23 @@ paths = [
 [haproxy17]
 plan_path = "haproxy17"
 paths = [
-  "haproxy/*"
+  "haproxy/*",
+  "haproxy16/*"
```

Comment:
This may need to change to another version if 16 gets dropped due to age. It might be wise to source 19 so it uses the last 1.x version that will ever be removed from this repo.

Reply:
If we're using the existing plans that `source` other plans as a reference, then I believe @james-stocks is right here, because they always seem to reference the latest version, and not the oldest version.

What do you think @predominant ?

Reply:
At the moment, it doesn't matter which one we source. :) Changing to 1.9

Reply:
Done.